speaking-portal-project/src/index.ts

```diff
@@ -31,15 +31,15 @@ const recognizer_options: string[] = ['English (U.S.)', 'French', 'Japanese','En
 
 // Main Function
 async function main() {
-    // TODO: Change language_selection based on desired language
+    // TODO: Change case number during peer testing
     // English = 0, French = 1, Japanese = 2
```

Comment:
```suggestion
    // English = 0, French = 1, Japanese = 2, Wrong file = 3, Corrupted file = 4, Unique files = 5|6|7|8
```

Reply:
Does case 3 have a pointer to an empty string ?